spec/controllers/video_papers_controller_spec.rb

```diff
@@ -181,12 +181,6 @@
         @user2 = FactoryBot.create(:user)
       end
 
-      it "should not fail in html" do
-        get :unshare, {:id => @paper.id, :user_id => @user2.id}
-        expect(response.status).to eq 302
-        expect(response).to redirect_to share_video_paper_path(@paper)
-      end
-
```

Comment:
Why is this test removed?

Reply:
I added that test in the last go around but we never call unshare in html only in xhr.  I just removed it to clean up the code when I was fighting with the cucumber unshare test.